src/external/onerep.js

```diff
@@ -137,6 +141,11 @@ export async function activateProfile(profileId) {
     method: "PUT",
   });
   if (!response.ok) {
+    const result = await response.json();
+    if (result.message === "Profile is already active") {
+      // TODO is there an advantage to checking first or is this that a wasted API call?
```

Comment:
Note that this will throw an unrelated error if the response body does not include a `.message` property. And given that I think we generally would not expect `activateProfile` to be called multiple times for the same profile (i.e. it's a genuine error condition), I think we _do_ want to log it?

Reply:
Hm yes I wasn't sure whether to throw an error or continue... I also considered making a call to check if the user was already activated, but this seems simpler and doesn't use an extra API call.

I think the only time this would matter is if the profile was activated but calling out-out failed for some reason, it's not harmful to try again. However it shouldn't be necessary.